ansible/roles/common/tasks/utils.yml

```diff
@@ -21,4 +21,5 @@
         'iftop',
         'ncdu',
         'nethogs',
-        'vnstat']
\ No newline at end of file
+        'vnstat',
+        ]
```

Comment:
1. Can you please push another commit to add the final newline? (We will definitely squash small PRs and so it will be very clean.)

2. It seems that your editor has not been configured to support `.editorconfig` (or else I guess the patch would have added the final newline)?https://github.com/picoCTF/picoCTF/blob/ab8165baf225f9ec65e287e0293eef361e82279a/.editorconfig#L14
P.S. If my guess is right, can you please enable support for `.editorconfig` in your editor(s)? Thanks!

Reply:
Added a newline in 2f60238f0c740d76023db83943ee73cf763b4f8a. You're right, I didn't have my `.editorconfig` extension added yet. It is now set up.